providers/dns/inwx/inwx.go

```diff
@@ -182,9 +183,11 @@ func (d *DNSProvider) CleanUp(domain, token, keyAuth string) error {
 
 	var lastErr error
 	for _, record := range response.Records {
-		err = d.client.Nameservers.DeleteRecord(record.ID)
-		if err != nil {
-			lastErr = fmt.Errorf("inwx: %w", err)
+		if record.Content == strconv.Quote(challengeInfo.Value) {
```

Comment:
are you sure the record content has quotes?

Reply:
You're right, I checked the API Docs and it is noted in double-quoted strings

Reply:
OK, can you run lego with your PR and check that the TXT records are deleted?

Reply:
I have just tested the implementation successfully. The reply did indeed not contain quotes.